source/codegen/metadata/nidaqmx/functions.py

```diff
@@ -6014,7 +6049,7 @@
                 'name': 'stateArray',
                 'size': {
                     'mechanism': 'passed-in-by-ptr',
-                    'value': 'arraySize'
+                    'value': 'arraySizePtr'
```

Comment:
This changes the field name in nidaqmx.proto from `array_size` to `array_size_ptr`, which breaks source compatibility.

None of the other field names in `.proto` files end in `_ptr`:
```
PS C:\dev\grpc-device> rg "_ptr" -t proto --type-add proto:*.proto
generated\nirfmxnr\nirfmxnr.proto
2894:  repeated nidevice_grpc.NIComplexNumberF32 pdsch_ptrs_constellation = 2;
2994:  repeated nidevice_grpc.NIComplexNumberF32 pusch_ptrs_constellation = 2;
```

We may need a special case in scrapigen to fix this.

Reply:
Understood @bkeryan . We can add some special case in scrapigen to fix this and the other similar changes that happened.

As these changes happened because of the version update for the source files. Shall we handle these changes and create a separate PR in scrapigen ?

Any thoughts?

Reply:
Please fix this. Please don't check unwanted compatibility breakage into main.

Reply:
I think there's something like `grpc_name` to give a different name in grpc only, if that's the desired result.

Reply:
And in scrapigen, RFmx does it generally for param names like "instrumentHandle" that we just want to be "instrument" in grpc. You can see the scrapigen code for that [here](https://github.com/ni/grpc-device-scrapigen/blob/51d2dccebe162ed1476457003187870c6389f25d/src/scrapigen/rfmxnr/transformfunctions.py#L77). You should be able to do something similarly for this parameter name.

Reply:
Added a fix in https://github.com/ni/grpc-device-scrapigen/pull/127/commits/2ff22d5b025bf6287e43aff545216a1ba34eca4a to remove Ptr from the value and name